priority_variables_transform/MESA-ingest/potassium.yaml

```diff
@@ -9,6 +9,8 @@
           value_mappings:
             '3': MESA CLASSIC EXAM 3
             '4': MESA CLASSIC EXAM 4
+        age_at_observation:
+          expr: case((phv00191192 == 4, {phv00086727} * 365), phv00191192 == 3, {phv00086259} * 365)
```

Comment:
This one has a new and subtle problem so I'll just fix it here:
```
expr: case(({phv00191192} == 4, {phv00086727} * 365), ({phv00191192} == 3, {phv00086259} * 365))
```

Reply:
fixed